docs/dag-serialization.rst

```diff
@@ -58,11 +58,16 @@ Add the following settings in ``airflow.cfg``:
     [core]
     store_serialized_dags = True
     min_serialized_dag_update_interval = 30
+    min_serialized_dag_fetch_interval = 10
```

Comment:
You don't need to set this -- it is already the default.

Reply:
Added a comment in a line above (in https://github.com/apache/airflow/pull/9851/commits/02856d63a5d500788c8aab7c49a235b036800364)